source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizerTests/ADCIOSVisualizerTests.mm

```diff
@@ -210,7 +210,7 @@ - (void)parseAndRenderDFS:(NSString *)rootPath FileManager:(NSFileManager *)fMan
                     [self assertRendering:renderResult fileName:fileName];
                 }
                 @catch (NSException *exception) {
-                    XCTMutableIssue *issue = [[XCTMutableIssue alloc] initWithType:XCTIssueTypeAssertionFailure compactDescription:[NSString stringWithFormat:@"Rendering Exception in %@", fileName]];
+                    XCTMutableIssue *issue = [[XCTMutableIssue alloc] initWithType:XCTIssueTypeAssertionFailure compactDescription:[NSString stringWithFormat:@"Rendering Exception in %@ with %@ \nStack Trace:%@", fileName, exception, exception.callStackSymbols]];
```

Comment:
Does the test case itself have problem? Here is the new output
  testParsingAndRenderingAllCards, Rendering Exception in IconsInSomeActions.json with Cannot remove an observer <ACRView 0x7feca3319180> for the key path "image" from <UIImageView 0x7feca3327170> because it is not registered as an observer.

Reply:
/Users/runner/work/1/s/source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizerTests/ADCIOSVisualizerTests.mm:213
  ```
                @catch (NSException *exception) {
                    XCTMutableIssue *issue = [[XCTMutableIssue alloc] initWithType:XCTIssueTypeAssertionFailure compactDescription:[NSString stringWithFormat:@"Rendering Exception in %@ with %@ \nStack Trace:%@", fileName, exception, exception.callStackSymbols]];
                    [self recordIssue:issue];
  ```

Reply:
you're seeing the new input because of this PR.

Reply:
it's fixed in [here](https://github.com/microsoft/AdaptiveCards/pull/6878)